packages/v4/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -0,0 +1,111 @@
+---
+id: Release highlights
+title: Release highlights
+section: get-started
+---
+
+import './get-started.css';
+
+# 2023.06
```

Comment:
Not sure why we are calling this 2023.06. Isn't this intended to be PatternFly 5.0?

Reply:
Sure! Will future releases resemble the 2023.xx format or PatternFly 5.0.x? Admittedly I just tried to guess based on the current website